src/main/webapp/i18n/de/aboutUs.json

```diff
@@ -3,31 +3,104 @@
         "title": "Artemis: Interaktives Lernen mit individuellem Feedback",
         "feature": "Haupteigenschaften",
         "exercises": {
-            "title": "Artemis unterstützt folgende Aufgabentypen",
-            "detail": {
-                "programmingExercises": "Programmierübungen mit Versionskontrolle und automatischer Bewertung anhand von Testfällen und kontinuierlicher Integration",
-                "quiz": "Quizübungen mit Multiple-Choice-, Drag & Drop- und Kurzantwort-Quizfragen",
-                "modelingExercises": "Modellierungsübungen mit halbautomatischer Bewertung unter Verwendung von Machine Learning Konzepten",
-                "textExercises": "Textübungen mit manueller (und experimenteller halbautomatischer) Bewertung",
-                "uploadExercises": "Übungen zum Hochladen von Dateien mit anschießender manueller Bewertung"
+            "programming": {
+                "text": "<a href=\"https://docs.artemis.ase.in.tum.de/user/exercises/programming/\">Programmierübungen</a> mit Versionskontrolle, automatischer individueller Rückmeldung (und Bewertung) auf der Grundlage von Testfällen und statischer Code-Analyse (durchgeführt mit kontinuierlicher Integration).",
```

Comment:
I would like to strike links like that. They do not belong in a language file!
In general, we should avoid HTML tags but one could argue for design tags like `<b></b>`. But we should move links out and just leave the Link Replacement and the rest of the text in the language file.

In the TS file you could then also set a constant to `https://docs.artemis.ase.in.tum.de/` and use this constant in the HTML file. This way we can easily replace the URL if it would change.

Reply:
Edit - i am not sure if this is possible since those tags are only rendered if we use `[jhiTranslate]` and not rendered if we use the translate pipe.

Reply:
Thats the reason why those tags are added to the tranlations

Reply:
Should be solved now :+1: